lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -1035,7 +1043,7 @@ sub xml_search
     if (DBDefs->SEARCH_ENGINE eq 'LUCENE') {
         $search_url_string = "http://%s/ws/$version/%s/?query=%s&offset=%s&max=%s&fmt=xml";
     } else {
-        $search_url_string = "http://%s/%s/select?q=%s&start=%s&rows=%s&wt=mbxml&fl=score";
+        $search_url_string = "http://%s/%s/edismax?q=%s&start=%s&rows=%s&wt=mbxml";
```

Comment:
same doubt here, although earlier it was using the lucene syntax

Reply:
`edismax` seems correct given the `$query` it constructs above.